File: Button TextView Demo/MainActivity.java
package com.example.demo2;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    //Declare variables/objects
    Button button1;
    TextView lbl_message;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //define/assign
        button1 = (Button) findViewById(R.id.button1);
        lbl_message = (TextView) findViewById(R.id.lbl_message);


        button1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                lbl_message.setText("Computer Science");
            }
        });


    }
}

File: Button TextView Demo/MainActivity.kt
package com.example.demo2

import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var button1: Button
    private lateinit var lblMessage: TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button1 = findViewById(R.id.button1)
        lblMessage = findViewById(R.id.lbl_message)
        button1.setOnClickListener {
            lblMessage.text = "Computer Science"
        }
    }
}
